java/src/jmri/jmrit/dispatcher/ActivateTrainFrame.java

```diff
@@ -1136,7 +1135,7 @@ private TrainInfo dialogToTrainInfo() {
         info.setLoadAtStartup(loadAtStartupBox.isSelected());
         info.setAllocateAllTheWay(false); // force to false next field is now used.
         if (allocateAllTheWayRadioButton.isSelected()) {
-            info.setAllocationMethod(999);
+            info.setAllocationMethod(-1);
         } else if (allocateBySafeRadioButton.isSelected()) {
             info.setAllocationMethod(0);
```

Comment:
You might want to use the constant ActiveTrain.ALLOCATE_BY_SAFE_SECTIONS instead of 0 for this.